src/lookupDataProvider.js

```diff
@@ -7,6 +7,12 @@ class LookupDataProvider extends Component {
         lookupData: null
     }
 
+    async componentDidMount() {
+        const data = await lookupTablesService.getAllLookupData();
+        this.setState({
+            lookupData: data.data
+        });
+    }
     render() {
         return (
```

Comment:
metoda `loadLookupData()` este stale ostala v datach, ktore posiela provider. je to nutne? mam pocit, ze nie :)

Reply:
netreba.. dobry postreh